### DISCUSSION TITLE
Problem with interfacing OSQP

### DISCUSSION THREAD

**Discussion by RudolfReiter** [casadi/casadi#3136] [Q&A]
Dear all!
I get the following problem in python on a LINUX machine when interfacing **OSQP** by using the opti-stack and the "conic" option from casadi 3.6.1.

ERROR in validate_data: P does not have dimension n x n with n = 113602864
ERROR in osqp_setup: Data validation returned failure
ERROR in validate_data: P does not have dimension n x n with n = 208
ERROR in osqp_setup: Data validation returned failure

When I use casadi 3.5.5, i do not get this error. However, the stats field is empty (all timings are zero) and second, the solution is quite off the nominal one, which I get with ipopot, gurobi or qpoases.

QPOASES with conic and ipopt when formulated as nlp, **work fine** though.

Best regards,
Rudolf

**Comment by jgillis** [comment]
Dear Rudolf,

I cannot reproduce this. Do you have a testcase to share?

**Comment by RudolfReiter** [comment]
Dear Joris, thanks for answering!
It is unfortunately part of confidential work but I try to create a toy problem.
Best,
Rudolf
